### ISSUE TITLE
META - Ecosystem terrain standardization

### ISSUE THREAD

**Issue by Ryanf55** [ROS-Aerial/aerial_robotic_landscape#42]
# Design Goals

* Use the same terrain data across all devices in the ecosystem
  * Autopilot (PX4, ArduPilot, Etc)
  * Companion Computer (grid_map_geo)
  * Ground Station (QGC, MAVProxy, Mission Planner)
  * Simulator (Gazebo)
  * Developer visualization tool (rviz, foxglove)
* Support loading terrain from an online terrain server
  * ArduPilot SRTM
  * AirMap
* Reduce network usage by maintaining a local cache of terrain data
  * Only download tiles once from online terrain server to devices in field
  * Cache terrain on the companion computer and/or autopilot to reduce GCS-Vehicle network congestion

**Comment by knmcguire** [comment]
Hi! shall I transfer this to the community repository? It doesn't seem like it is an addition for the aerial robotic landscape documentation (at least for now) but more of a goal of the outdoor planning subcommittee.

**Comment by Ryanf55** [comment]
> Hi! shall I transfer this to the community repository? It doesn't seem like it is an addition for the aerial robotic landscape documentation (at least for now) but more of a goal of the outdoor planning subcommittee.

Right, yes please! I created it in the outdoor 3D planning project and it looks like this was the default.
